pandas/core/frame.py

```diff
@@ -2282,7 +2287,7 @@ def info(self, verbose=None, buf=None, max_cols=None, memory_usage=None,
         text_col     5 non-null object
         float_col    5 non-null float64
         dtypes: float64(1), int64(1), object(1)
-        memory usage: 200.0+ bytes
+        memory usage: 248.0+ bytes
```

Comment:
You are changing this in several places. Is this related to this PR? 

(it seems that your changes are correct, but then we would prefer to have those changes in a separate PR if they are not related to the deprecation of to_sparse/to_dense)